src/main/java/eu/eutampieri/catacombs/model/map/TileMapFactoryImpl.java

```diff
@@ -41,145 +42,172 @@ public boolean equals(Object obj) {
 				return false;
 			return true;
 		}
-	final int x;
-	  final int y;
-	  Point(int x, int y) {this.x=x;this.y=y;}
-	  int dist(Point o) {
-		  return Math.abs(x-o.x)+Math.abs(y-o.y);
-	  }
-	private TileMapFactoryImpl getEnclosingInstance() {
-		return TileMapFactoryImpl.this;
-	}
+
+		final int x;
+		final int y;
+
+		Point(int x, int y) {
+			this.x = x;
+			this.y = y;
+		}
+
+		int dist(Point o) {
```

Comment:
We're in javaland, you could have used Point other

Reply:
I could have also used thisIsTheCoordinateXThatRepresentsTheDistanceFromTheLeftBoundary, but I prefer not